app/code/Magento/Review/Model/ReviewManagement.php

```diff
@@ -0,0 +1,235 @@
+<?php
+namespace Magento\Review\Model;
+
+use Magento\Review\Api\ReviewInterface;
+use Magento\Review\Model\Review;
+use Magento\Review\Model\ResourceModel\Review\CollectionFactory;
+
+class ReviewManagement implements \Magento\Review\Api\ReviewInterface
```

Comment:
This is being imported but you're using the full namespace here anyway, can you change it to ReviewInterface